src/map/pc.cpp

```diff
@@ -2427,14 +2427,16 @@ void pc_clean_skilltree(struct map_session_data *sd)
 uint64 pc_calc_skilltree_normalize_job_sub( struct map_session_data *sd ){
 	int skill_point = pc_calc_skillpoint( sd );
 
-	if( sd->class_ & MAPID_SUMMONER ){
-		// Novice's skill points for basic skill.
+	if( sd->class_ & MAPID_SUMMONER || sd->class_ & MAPID_SPIRIT_HANDLER ){
```

Comment:
The change in the condition is useless as MAPID_SUMMONER is part of MAPID_SPIRIT_HANDLER.
Applies to all others below as well.